diff --git a/include/configs/sun7i.h b/include/configs/sun7i.h
index 56101a9..b5b6f40 100644
--- a/include/configs/sun7i.h
+++ b/include/configs/sun7i.h
@@ -24,6 +24,12 @@
 #define CONFIG_ARMV7_SECURE_BASE	SUNXI_SRAM_B_BASE
 #define CONFIG_TIMER_CLK_FREQ		24000000
 
+#define	CONFIG_BOOTCOMMAND	"ext2load mmc 0:1 48000000 /boot/vmlinuz; bootz 48000000"
+#define	CONFIG_IPADDR		192.168.0.104
+#define	CONFIG_NETMASK		255.255.252.0
+#define	CONFIG_SERVERIP		192.168.0.99
+#define	CONFIG_BOOTARGS		"console=ttyS0,115200 rootfstype=ext4 root=/dev/mmcblk0p1 earlyprintk rootwait"
+
 /*
  * Include common sunxi configuration where most the settings are
  */
@@ -31,4 +37,11 @@
 
 #define CONFIG_MACH_TYPE	(4283 | ((CONFIG_MACH_TYPE_COMPAT_REV) << 28))
 
+#undef	BOOTENV
+#define	BOOTENV
+
+#undef CONFIG_EXTRA_ENV_SETTINGS
+#define CONFIG_EXTRA_ENV_SETTINGS
+
+
 #endif /* __CONFIG_H */
